tests/integration/Core/Repository/ContentService/CopyTranslationsFromPublishedVersionTest.php

```diff
@@ -0,0 +1,115 @@
+use eZ\Publish\Core\Repository\Values\Content\ContentUpdateStruct;
+use Ibexa\Tests\Integration\Core\RepositoryTestCase;
+
+/**
+ * @covers \eZ\Publish\API\Repository\ContentService
+ */
+final class CopyTranslationsFromPublishedVersionTest extends RepositoryTestCase
+{
+    private const ENG_LANGUAGE_CODE = 'eng-GB';
+    private const GER_LANGUAGE_CODE = 'ger-DE';
+    private const US_LANGUAGE_CODE = 'eng-US';
+    private const CONTENT_TYPE_IDENTIFIER = 'custom';
+
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\Exception
+     */
+    public function testCopyTranslationsFromPublishedVersionCopiesEmptyValues(): void
+    {
+        $this->createContentType();
+
+        $contentService = self::getContentService();
+        $contentTypeService = self::getContentTypeService();
+        $locationService = self::getLocationService();
+
+        // Creating and publishing content in eng-GB language
+        $contentType = $contentTypeService->loadContentTypeByIdentifier(self::CONTENT_TYPE_IDENTIFIER);
+        $mainLanguageCode = self::ENG_LANGUAGE_CODE;
+        $contentCreateStruct = $contentService->newContentCreateStruct($contentType, $mainLanguageCode);
+        $contentCreateStruct->setField('title', 'Test title');
+
+        $contentDraft = $contentService->createContent(
+            $contentCreateStruct,
+            [
+                $locationService->newLocationCreateStruct(2),
+            ],
+        );
+        $publishedContent = $contentService->publishVersion($contentDraft->getVersionInfo());
+
+        // Creating a draft and publishing in ger-DE language with empty 'title' field
+        $gerDraft = $contentService->createContentDraft($publishedContent->contentInfo);
+        $usDraft = $contentService->createContentDraft($publishedContent->contentInfo);
+
+        $contentUpdateStruct = new ContentUpdateStruct([
+            'initialLanguageCode' => self::GER_LANGUAGE_CODE,
+        ]);
+        $contentUpdateStruct->setField('title', null);
+        $gerContent = $contentService->updateContent($gerDraft->getVersionInfo(), $contentUpdateStruct);
+        $contentService->publishVersion($gerContent->getVersionInfo(), [self::GER_LANGUAGE_CODE]);
+
+        // Creating a draft and publishing in eng-US language with empty 'title' field
+        $contentUpdateStruct = new ContentUpdateStruct([
+            'initialLanguageCode' => self::US_LANGUAGE_CODE,
+        ]);
+        $contentUpdateStruct->setField('title', null);
+        $usContent = $contentService->updateContent($usDraft->getVersionInfo(), $contentUpdateStruct);
+        $publishedUsContent = $contentService->publishVersion($usContent->getVersionInfo(), [self::US_LANGUAGE_CODE]);
+
+        $gerFieldValueInUsContent = $publishedUsContent->getField('title', self::GER_LANGUAGE_CODE)
+            ->getValue()
+            ->text;
+
+        self::assertSame('', $gerFieldValueInUsContent);
```

Comment:
1. `->text` is a polymorphic call and this will most likely make PHPStan fail in ibexa/core without extra assertions.
2. Let's add sanity test for eng-US field value as well.

1 & 2:
```suggestion
        $usFieldValueInUsContent = $publishedUsContent->getField('title', self::US_LANGUAGE_CODE)->getValue();
        self::assertInstanceOf(TextLine\Value::class, $usFieldValueInUsContent);
        self::assertSame('', $usFieldValueInUsContent->text);

        $gerFieldValueInUsContent = $publishedUsContent->getField('title', self::GER_LANGUAGE_CODE)->getValue();
        self::assertInstanceOf(TextLine\Value::class, $gerFieldValueInUsContent);
        self::assertSame('', $gerFieldValueInUsContent->text);
```